nservicebus/azure-service-bus/index.md

```diff
@@ -11,37 +11,37 @@ related:
  - samples/azure/azure-service-bus
 ---
 
-In some environments it is not possible or recommended to rely heavily on distributed transactions to ensure reliability and consistency. Therefore in environments such as very large cloud networks or hybrid networks using MSMQ is not the best idea. In those scenarios a good alternative is Azure Service Bus.
+[Azure Service Bus (ASB)](https://azure.microsoft.com/en-us/services/service-bus/)  is a messaging service hosted on the Azure platform, that allows for exchanging messages between various applications in a loosely coupled fashion. ASB Queues offer FIFO (First In, First Out) guaranteed message delivery, and support a range of standard protocols (REST, AMQP, WS*) and APIs (to put/pull messages on/off the queue). ASB Topics deliver messages to multiple subscribers and allow to use fan-out pattern to deliver messages to downstream systems.
 
-NOTE: As part of the Azure support for NServiceBus, one can choose between two transports provided by the Azure platform Azure Storage Queues and Azure Service Bus. Each of these two options has separate features, capabilities, and usage characteristics. A detailed comparison and discussion of when to select which is beyond the scope of this document. To help decide which option best suits the application's needs, review the Azure article [Azure Queues and Azure Service Bus Queues - Compared and Contrasted](https://azure.microsoft.com/en-us/documentation/articles/service-bus-azure-and-service-bus-queues-compared-contrasted/).
+NServiceBus is an abstraction over ASB. It takes advantage of ASB's built-in features, such as message deduplication and deferred messages. It also provides higher-level, convenient API for programmers on top of ASB.
```

Comment:
(it is valid)

You meant, why I haven't changed the text yet, right?

Because I still need to go through these comments, tackled the ones on the issue conversation so far.